basics/slices.md

```diff
@@ -0,0 +1,87 @@
+# Slices
+
+Slices sind Objekte des Types `T[]` für einen beliebigen Typen `T`.
+Slices sind eine Untermenge (Sicht) oder die gesamte Menge
+eines Arrays.
+**Dynamische Arrays sind Slices.**
+
+Ein Slice beinhaltet zwei Eigenschaften: (1) einen Zeiger auf das erste Element und
+(2) die Länge des Slices:
+
+    T* ptr;
+    size_t length; // uint auf 32-bit, ulong auf 64-bit (Siehe http://tour.dlang.org/tour/en/basics/basic-types)
+
+### Erstellung eines neuen Slices mit `new`
+
+Bei der Erstellung eines dynamischen Arrays wird ein Slice zum
+erstellten Speicher zurückgegeben.
+
+    auto arr = new int[5];
+    assert(arr.length == 5); // Speicheraddresse in arr.ptr
+
+Der eigentliche Speicher wird von der automatischen Speicherbereinigung
+(Garbage Collection) verwaltet. Der Slice stellt hier eine Menge an Elementen dar.
+
+### Erstellung eines Slices durch bestehenden Speicher
+
+Der Slice-Operator kann auch benutzt werden, um auf bestenden Speicher zu zeigen.
+Der Slice-Operator kann auf ein anderes Slice, auf statische Arrays, auf
+Strukturen und Klassen welche `opSlice` implementieren und ein paar andere Objekte
+angewendet werden.
+
+Beim Ausdruck `arr[Start .. Ende]` gibt der Slice-Operator die Elemente von `arr`
+anfangend bei `Start` bis aus­schließ­lich `Ende` als Untermenge zurück.
+
+    auto newArr = arr[1 .. 4]; // Index 4 wird NICHT mit einbezogen
+    assert(newArr.length == 3);
+    newArr[0] = 10; // ändert newArr[0] und somit arr[1]
+
+Solche Slices erstellen keine Kopie vom Speicher, sondern geben nur eine
+Untermenge (Sicht) des Arrays an. Wenn kein Slice mehr zum Speicher oder zu einer
+Untermenge zeigt, wird der Speicher bei der Garbage Collection gelöscht.
+
+Mit Slices kann speichereffizienter Code geschrieben werden, welcher mit nur einem
+Block an Speicher arbeitet und Sichten auf Unterbereiche des Speicherblockes zur
+Vermeidung neuer Speicherallokationen benutzt.
+
+Im vorherigen Kapitel wurde angesprochen, dass `arr[$]` eine kurze Form von
+`arr[arr.length]` ist und damit versucht das Element nach dem Ende des Slices
+zu bekommen. Das würde ein `RangeError` werfen, wenn `bounds-checking` nicht
+deaktiviert wurde.
```

Comment:
Complain here: https://github.com/dlang-tour/english/blob/master/basics/slices.md#getting-a-slice-to-existing-memory